### ISSUE TITLE
ZIP archive broken

### ISSUE THREAD

**Issue by julmot** [MinhasKamal/DownGit#14]
Download this link:

https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/julmot/mark.js/tree/master/dist

The resulting folder is empty for me.

**Comment by MinhasKamal** [comment]
@julmot Please check if the problem persists.

**Comment by julmot** [comment]
@MinhasKamal Yes, "the file '' can not be opened as archive" when unzipping wihth 7zip.

**Comment by MinhasKamal** [comment]
@julmot I have successfully downloaded and unzipped [the directory](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/julmot/mark.js/tree/master/dist), not once, rather several times.

**Comment by julmot** [comment]
No clue why your tests were successfully. Clicking on this link results in an empty zip file:
https://minhaskamal.github.io/DownGit/#/home?url=https:%2F%2Fgithub.com%2Fjulmot%2Fmark.js%2Ftree%2Fmaster%2Fdist

**Comment by MinhasKamal** [comment]
I have clicked your link, works for me just fine.

**Comment by julmot** [comment]
You're a good developer ;-) Closing an issue without asking one simple question, for example browser, operation system, version and so one. Just closing the issue won't make it disappear. OMG!

**Comment by MinhasKamal** [comment]
Sorry :(
Got no response from you, so I thought you may have succeeded in downloading.

**Comment by Z0ckerchris** [comment]
@julmot If you have some sort of script-blocker, try disabling it. Or try making exceptions for GitHub URLs.
That solved the problem for me.
